back-end/src/test/java/admin/gpuserver/domain/GpuBoardTest.java

```diff
@@ -0,0 +1,78 @@
+package admin.gpuserver.domain;
+
+import admin.gpuserver.exception.GpuServerServiceException;
+import admin.lab.domain.Lab;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import static org.assertj.core.api.Assertions.assertThatCode;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+public class GpuBoardTest {
+    private GpuServer gpuServer;
+
+    @BeforeEach
+    void setUp() {
+        Lab lab = new Lab("랩1");
+        gpuServer = new GpuServer("GPU서버1", false, 1024L, 1024L, lab);
+    }
+
+    @DisplayName("생성 테스트 - 정상")
+    @Test
+    void 생성() {
+        assertThatCode(() -> new GpuBoard(false, 1000L, "모델1", gpuServer))
+                .doesNotThrowAnyException();
+    }
```

Comment:
```suggestion
    @DisplayName("생성 테스트 - 정상")
    @Test
    void 생성() {
        GpuBoard gpuBoard = new GpuBoard(false, 1000L, "모델1", gpuServer));
         assertThat(gpuBoard).isNotNull();
    }
```

실세로 생성이 되었는지를 확인해보는 것은 어떨까요?

에러가 나지 않는다 = 생성이 되었다 가 비슷하지만 동치는 아닐것 같은 개인적인 생각입니다.

Reply:
저는 방법의 차이라고 생각합니다.
위는 생성 과정이 올바르게 진행되는지에 포커스를 맞춘것이고,
아래는 객체가 생성되었는지에 포커스를 맞추는 것이라 생가합니다.